test/fixtures/block.ex

```diff
@@ -0,0 +1,94 @@
+defmodule Fixtures.Block do
```

Comment:
TODO: change integers for random integers?

Reply:
Could be, but it's something we could work on as a follow up.

Reply:
If it's just generating random numbers, you can just use `:rand.uniform/1`. Should be a small change, if you're interested.